### DISCUSSION TITLE
`replacements:containerbase` contains duplicate keys

### DISCUSSION THREAD

**Discussion by mueller-ma** [renovatebot/renovate#30470] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

None

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Latest main

### Please tell us more about your question or problem

Two rules contain `matchPackageNames` twice: https://github.com/renovatebot/renovate/blob/main/lib/data/replacements.json#L172-L182

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Thanks, https://github.com/renovatebot/renovate/pull/30471
